src/content/whats-new/2023/04/whats-new-04-06-alert-condition-form.md

```diff
@@ -1,8 +1,10 @@
 ---
 title: 'Streamline alert configuration and management'
-summary: 'Set up alerts faster and with more flexible configuration options.'
+summary: 'Set up alerts faster and with more flexible configuration options.’
+
 releaseDate: '2023-04-06'
```

Comment:
Can a PR reviewer change this release date to `'2023-04-19'` for us? Or would we need to fork his repo and push up a new PR?

Reply:
@nateborr I can change it for you.

Reply:
@nateborr I can change it for you.